cli/sign.go

```diff
@@ -0,0 +1,44 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/cmd/cosign/cli/sign"
+
+	// Loads OIDC providers
+	_ "github.com/sigstore/cosign/pkg/providers/all"
+)
+
+// Sign applies keyless OIDC signatures to sign UOR Collections
+func Sign(ctx context.Context, dest string) error {
```

Comment:
Since this will only be used within the cli package, I would prefer this be unexported. WDYT?

Reply:
i agree